### ISSUE TITLE
aria-expanded attribute has no matching role attribute

### ISSUE THREAD

**Issue by MattSidor** [AlaskaAirlines/auro-select#77]
## Describe the bug

In auro-select, the `<auro-dropdown>` element's [`aria-expanded`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) attribute indicates the dropdown's state, but is being flagged by axe-core 4.4.2 (Chrome extension) since the same element has no [`role`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) attribute applied when used in auro-select. According to the tool, the `aria-expanded` attribute will be ignored by browsers.

auro-combobox solved this issue by adding `role="combobox"` to the element. The appropriate role to be applied to `<auro-dropdown>` in auro-select is probably "listbox".

## To Reproduce

1. Go to https://auro.alaskaair.com/components/auro/select
1. Run [Axe DevTools extension](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)
1. Note violation of rule ID [`aria-allowed-attr`](https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr) within each `<auro-select>` element.

## Expected behavior

No Axe rule violations

## Desktop (please complete the following information):

 - OS: macOS
 - Browser: Chrome 
 - Version: 103

**Comment by MattSidor** [comment]
My team has been granted capacity to work on this issue during our current sprint, since it impacts our applications. I am working on this now.

**Comment by blackfalcon** [comment]
@MattSidor please update the label of the issue as `in progress` once you have actively started working on this. Also a `draft` PR is appreciated to see progress leading up to the final PR. 

Thanks for the support.

**Comment by MattSidor** [comment]
In working on this, I discovered that the changes should ideally be made in `<auro-dropdown>` and not `<auro-select>`. I'm going to open a new issue over there and I will have a PR up for it later today with demos. I'll be sure to link the new issue back here. Closing this issue.
